tests/queries.py

```diff
@@ -107,9 +107,11 @@ def get_blocks_and_layers(namespace, pod_name, find_fails=False):
 
 
 def get_layers(namespace, find_fails=True):
-    layers = get_all_msg_containing(namespace, namespace, "release tick", find_fails)
-    ids = [int(x.layer_id) for x in layers]
-    return ids
+    tick_msgs = get_all_msg_containing(namespace, namespace, "release tick", find_fails)
+    layers = defaultdict(int)
+    for msg in tick_msgs:
+        layers[msg.layer_id] += 1
+    return layers
 
```

Comment:
we can calculate max here and return it 
max = msg.layer_id if msg.layer_id > max else max

Reply:
We're not using the max. We're going from the highest layer down until we find one with enough miners, and return that.